packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -310,9 +310,13 @@ class RenderUiKitView extends RenderBox {
   UiKitViewController get viewController => _viewController;
   UiKitViewController _viewController;
   set viewController(UiKitViewController viewController) {
+    final bool needsSemanticsUpdate = _viewController.id != viewController.id;
```

Comment:
Move this below the assert to avoid the potential for a null-pointer exception.

Reply:
done